src/Common/CommonAssemblyInfo.cs

```diff
@@ -10,7 +10,6 @@
 [assembly: AssemblyCopyright("© Microsoft Corporation. All rights reserved.")]
 
 [assembly: ComVisible(false)]
-[assembly: CLSCompliant(true)]
```

Comment:
I don't know why we ever put this here. It just causes a lot of pain. In my case, the new http binding package assembly wasn't cls compliant, so it caused all kinds of warnings/errors when trying to use those types in this cls compliant assembly.